content/gv1.md

```diff
@@ -18,7 +18,7 @@ If you're here, you've discovered the QR code on our 100GBASE-CWDM4 module which
 ### Explore More and Get Involved
 - **Join SONIX:** Use the module you have recieved to connect to our internet exchange by [signing up](https://forms.gle/xJ5CXbSWk8snzgND9).
```

Comment:
Could you update this with a link to the join page instead?

Reply:
Done!